pkg/util/util.go

```diff
@@ -138,3 +138,13 @@ func HasKernelBootContainerImage(vmi *v1.VirtualMachineInstance) bool {
 func HasHugePages(vmi *v1.VirtualMachineInstance) bool {
 	return vmi.Spec.Domain.Memory != nil && vmi.Spec.Domain.Memory.Hugepages != nil
 }
+
+func IsReadOnlyDisk(disk *v1.Disk) bool {
+	isReadOnlyCDRom := disk.CDRom != nil && (disk.CDRom.ReadOnly == nil || *disk.CDRom.ReadOnly == true)
+	if isReadOnlyCDRom {
+		return true
+	}
+
+	isReadOnlyFloppy := disk.Floppy != nil && disk.Floppy.ReadOnly
```

Comment:
I think you can get rid of the floppy disk related logic because of #2016 - floppy disks are rejected

Reply:
I also asked myself the same question :)
Since floppy is still part of our API shouldn't we check this just-in-case? And if not - why is it still part of our API?
@fabiand maybe you can help answer this?

Reply:
@iholder-redhat I also think you can skip the floppy here since we drop any attempt to use a floppy in the validating webhook. I think it's a leftover and we can deprecate it from the API in a follow up PR

Reply:
done.